ci/requirements.txt

```diff
@@ -1,6 +1,2 @@
 pygarrayimage
-hypothesis<3.0.0
-fonttools==3.28.0
-reportlab<=3.1
-kiwisolver ; python_version <= "2.7"
-https://bitbucket.org/pyglet/pyglet/get/pyglet-1.1.4.zip ; python_version <= "2.7"
+pyglet
```

Comment:
Do we want to be picky about a version here?

Reply:
Don't know.  I would default to most recent, but it likely depends on what the GL backend needs.  I don't think its a fast-moving API.